routers/users.py

```
@@ -1,16 +1,66 @@
+    users = read_json(USERS_FILE)
+
+    # 이메일 중복 확인
+    if any(u["email"] == user.email for u in users):
+        raise HTTPException(
+            status_code=status.HTTP_409_CONFLICT,
+            detail="Email already registered"
+        )
+
+    # 새 유저 ID 생성
+    new_id = f"user_{len(users) + 1}"
+    now = datetime.now()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)